workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Jobs/CodegenJob.cs

```diff
@@ -62,7 +62,12 @@ public void Run()
                     fileSystem.CreateDirectory(fileInfo.DirectoryPath);
                 }
 
-                fileSystem.WriteToFile(fileInfo.CompletePath, entry.Value);
+                // Fix up line endings
+                var contents = entry.Value
+                    .Replace("\r\n", "\n")
+                    .Replace("\n", Environment.NewLine);
```

Comment:
maybe add a comment on why we don't do replacements on just `\r` here

+ internal changelog

Reply:
How does the release tool react to me adding to the changelog at this point?
I'm not sure why this needs a comment, `\r` isn't a thing.

Reply:
> How does the release tool react to me adding to the changelog at this point?

It doesn't care